apps/minifront/src/fetchers/assets.ts

```diff
@@ -6,7 +6,9 @@ import { penumbra } from '../penumbra';
 
 export const getAllAssets = async (): Promise<Metadata[]> => {
   const responses = await Array.fromAsync(penumbra.service(ViewService).assets({}));
-  return responses.map(getDenomMetadata);
+  return responses
+    .map(getDenomMetadata)
+    .toSorted((a, b) => Number(b.priorityScore) - Number(a.priorityScore));
```

Comment:
question: can we also apply this to the `groupAndSort.ts` file in the UI package?